content/concept-amendments.md

```diff
@@ -270,12 +317,14 @@ An address with a SignerList can disable the master key even if a regular key is
 
 | Amendment ID                                                     | Status    |
 |:-----------------------------------------------------------------|:----------|
-| 9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 | Released but not enabled |
+| 9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 | In development |
 
 Fixes an inconsistency in the way [transfer fees](concept-transfer-fees.html) are calculated between [OfferCreate](reference-transaction-format.html#offercreate) and [Payment](reference-transaction-format.html#payment) transaction types. Without this amendment, the holder of the issuances pays the transfer fee if an offer is executed in offer placement, but the initial sender of a transaction pays the transfer fees for offers that are executed as part of payment processing. With this amendment, the holder of the issuances always pays the transfer fee, regardless of whether the offer is executed as part of a Payment or an OfferCreate transaction. Offer processing outside of payments is unaffected.
 
 This Amendment requires the [Flow Amendment](#flow) to be enabled.
 
+**Note:** An incomplete version of this amendment was introduced in v0.33.0 and removed in v0.80.0. (It was never enabled.) Ripple plans to re-add the amendment when the code is stable enough.
```

Comment:
It's baffling to me, then, that [OwnerPaysFee is commented out and FlowCross isn't](https://github.com/ripple/rippled/blob/0.80.0/src/ripple/app/main/Amendments.cpp#L48-L58) in rippled v0.80.0. Might want to look into that?

Reply:
That's fair.  The amendments that are commented out are the ones that we don't run (by default) against our unit tests.  We want to run the unit tests against code that we think is likely to be on the MainNet soon.  I'm personally hoping that `FlowCross` will make the cut sometime soon.  But I'm not in control of that.  As I recall, `MultiSign` was sitting, ready, in the code base for about a year before we finally had the grit to enable it.  In the 0.90.0 release you'll see `Tickets` and `SHAMapV2` are also commented out since `SHAMapV2` is not moving fast and the implementation for `Tickets` needs work.

At any rate, `OwnerPaysFee` was specifically commented out because Mike made a great change so unit tests automatically pick up the latest amendments.  When he did that a bunch of unit tests failed because `OwnerPaysFee` was enabled.  Nothing terrible happened, just the expected balances didn't turn out the way they used to.  So the simplest thing to do was remove `OwnerPaysFee` from being enabled by default in the unit tests.  If the product folks decide that the time is right for `OwnerPaysFee`, then the first thing to do will be fixing up the unit tests.